build.sbt

```diff
@@ -45,7 +45,6 @@ scalacOptions ++= Seq(
   "-language:implicitConversions",
   "-language:postfixOps",
   "-unchecked",
-  "-Xfatal-warnings",
```

Comment:
Could you please add a todo here to put this option back once the deprecated method is removed?